ember-cli-build.js

```diff
@@ -96,6 +97,12 @@ module.exports = function (defaults) {
         plugins: [
           customFilePlugin('version.txt', config.x.version),
 
+          codecovWebpackPlugin({
+            enableBundleAnalysis: process.env.CODECOV_TOKEN !== undefined,
+            bundleName: 'codecrafters-frontend-bundle',
```

Comment:
```suggestion
            bundleName: 'client',
```

(https://docs.codecov.com/docs/bundle-names)